specs/experimental/fault-proof/stage-one/bond-incentives.md

```diff
@@ -58,3 +97,39 @@ are located on the same level as honest claims, but situated to its right (i.e.
 An honest challenger can always successfully dispute any sibling claims not positioned to the right of an honest claim.
 The leftmost payoff rule encourages such disputes, ensuring only one claim is leftmost at correct depths.
 This claim will be the honest one, and thus bond rewards will be directed exclusively to honest claims.
+
+## Fallback on System Failure
+
+In the case that a dispute game resolves incorrectly in stage one while the system is nascent, the honest challenger
+stands to lose a significant amount of funds for performing the correct actions. The `Guardian` role in the
+`SuperchainConfig` can enable safety mode, which flags the `FaultDisputeGame` and `PermissionedDisputeGame` to return
+bonds to their submitters. To allow the `Guardian` enough time to enable safety mode in the event that a game resolves
+incorrectly, the `FaultDisputeGame`'s `claimCredit` function requires waiting `BOND_PAYOUT_DELAY` after the game
+resolves before any credit can be claimed.
+
+This is reflected in the `claimCredit` function of the `FaultDisputeGame`:
+
+```solidity
+/// @notice Claim the credit belonging to the recipient address.
+/// @param _recipient The owner and recipient of the credit.
+function claimCredit(address _recipient) external {
+    // Don't allow the credit to be claimed until bond delay has expired.
+    if (block.timestamp < resolvedAt.raw() + BOND_PAYOUT_DELAY.raw()) revert BondDelayNotExpired();
+
+    // Remove the credit from the recipient prior to performing the external call.
+    uint256 recipientCredit;
+    if (SUPERCHAIN_CONFIG.fdgSafetyMode()) {
+        recipientCredit = bonds[_recipient];
+        bonds[_recipient] = 0;
+    } else {
+        recipientCredit = credit[_recipient];
+        credit[_recipient] = 0;
+    }
+
+    // Transfer the credit to the recipient.
+    (bool success,) = _recipient.call{ value: recipientCredit }(hex"");
+    if (!success) revert BondTransferFailed();
+}
+```
+
+This fallback mechanism will be removed in the stage two project.
```

Comment:
We should not make promises about future changes. Just need to document how things work at this stage.